1d_serial/1d_nonlocal_serial.cpp

```diff
@@ -0,0 +1,174 @@
+#include <hpx/hpx_init.hpp>
```

Comment:
Please add the copyright header with your name to this file. If you have copied part of this file from the HPX repo, please add the copyright header from there.